lib/wallet/wallet.js

```diff
@@ -1849,37 +1850,40 @@ class Wallet extends EventEmitter {
    * @param {Number} value
    * @param {Number} lockup
    * @param {Object} options
+   * @returns {{ bid: MTX; reveal: MTX;}} {bid, reveal}
    */
 
   async _createAuctionTxs(name, value, lockup, options) {
     const bid = await this._createBid(name, value, lockup, options);
 
+    const bidOuputIndex = bid.outputs.findIndex(o => o.covenant.isBid());
+    const bidCoin = Coin.fromTX(bid, bidOuputIndex, this.wdb.height);
```

Comment:
I think `height` should be `-1` for an unconfirmed coin.

Reply:
Wasn't sure what to set here, good to know